app/lib/admin/actions/actions.dart

```diff
@@ -116,6 +117,7 @@ final class AdminAction {
     publisherCreate,
     publisherDelete,
     publisherInfo,
+    publisherInviteMember,
```

Comment:
maybe we should call this `publisherMemberInvite` (with the name of the action also `publisher-member-invite`) so that it aligns with `publisherMembersList`?